Lib/fontTools/feaLib/parser.py

```diff
@@ -10,13 +10,15 @@
 import re
 
 
+
```

Comment:
I put the line back because you asked me. My editor does what I ask it :) So do you want that line in there or not?